client/pages/_app.js

```diff
@@ -8,6 +8,7 @@
 import "../styles/globals.css";
 import "../styles/bulma.css";
 import "tippy.js/dist/tippy.css";
+import "../styles/Accordion.module.css"
```

Comment:
CSS Modules shouldn't be imported globally (unless they have global marked classes, I think marked with `:`).
You should be able to delete this addition 👍